src/routine.cpp

```diff
@@ -24,6 +24,7 @@ namespace clblast {
 // The constructor does all heavy work, errors are returned as exceptions
 Routine::Routine(Queue &queue, EventPointer event, const std::string &name,
                  const std::vector<std::string> &routines, const Precision precision,
+                 std::initializer_list<const char *> source_pct,
```

Comment:
OK, I understand your remark now. So then nevermind some of the comments I made. Indeed, GitHub can be a tricky with showing the comments, especially this one since it is hidden in history of an outdated change.